lib/buildkite/test_collector/uploader.rb

```diff
@@ -47,6 +47,7 @@ def self.upload(data)
           end
         rescue StandardError => e
           $stderr.puts "#{Buildkite::TestCollector::NAME} #{Buildkite::TestCollector::VERSION} experienced an error when sending your data, you may be missing some executions for this run."
+          $stderr.puts e
```

Comment:
```suggestion
          $stderr.puts e
          $stderr.puts "#{Buildkite::TestCollector::NAME} #{Buildkite::TestCollector::VERSION} experienced an error when sending your data, you may be missing some executions for this run."
```
Just moving the error above the log line so it is more like a stack trace